hieradata/common.yaml

```diff
@@ -781,6 +781,14 @@ govuk_jenkins::config::user_permissions:
 
 govuk::node::s_logging::apt_mirror_hostname: "%{hiera('apt_mirror_hostname')}"
 
+govuk_mtail::enabled: true
+govuk_mtail::extra_args: "-log_dir /var/log/mtail -graphite_prefix %{::fqdn_metrics}."
+govuk_mtail::graphite_host_port: 'graphite:2003'
+govuk_mtail::logs:
+  - '/var/log/nginx/*-access.log'
+govuk_mtail::manage_repo_class: true
```

Comment:
if the default is `true` in common.yaml, make it true by default in the manifest?

Reply:
I made it true directly when we implement the 'mtail' class